config/jobs/kubernetes/sig-testing/kubetest-canaries.yaml

```diff
@@ -75,11 +75,7 @@ periodics:
       args:
       - --deployment=node
       - --gcp-zone=us-west1-b
-        # node_e2e doesn't support passing an image-family on the CLI, only via
-        # a file, so here we manually specify an image to run tests against.
-        # To find the latest cos-97-lts image run:
-        #   gcloud compute images describe-from-family cos-97-lts --project cos-cloud --format='value(name)'
-      - --node-args=--images=cos-97-16919-29-21 --image-project=cos-cloud
+      - --node-args=--image-config-file=/workspace/test-infra/jobs/e2e_node/containerd/image-config.yaml
```

Comment:
@bart0sh i think the `canary` in the name implies that it is testing a new image. So we'll need a new image-config that uses the fresh image.

cc @SergeyKanzhelev

Reply:
@dims thank you for the review. I've added a separate image config to use `cos-beta`(COS) and `pipeline-1-28`(Ubuntu) image families. It should automatically use new images I guess.